external_modules/total_energy_module/build_total_energy_module.sh

```diff
@@ -38,9 +38,16 @@ modobjs="$(find $root_dir/Modules/ -name "*.o")"
 # Libraries from the external FoX project
 foxlibs="-L$root_dir/FoX/lib -lFoX_dom -lFoX_utils -lFoX_wcml -lFoX_wkml -lFoX_wxml -lFoX_common -lFoX_sax -lFoX_common -lFoX_fsys"
 
-project_lib_folders=" -L$root_dir/Modules -L$root_dir/KS_Solvers -L$root_dir/FFTXlib -L$root_dir/LAXlib -L$root_dir/UtilXlib -L$root_dir/dft-d3"
-project_libs="-lqemod -lks_solvers -lqefft -lqela -lutil -ldftd3qe"
-project_inc_folders="-I$root_dir/Modules -I$root_dir/FFTXlib -I$root_dir/LAXlib -I$root_dir/KS_Solvers -I$root_dir/UtilXlib"
+# LF: For some reason I cannot make the linker find the XClib if I don't rename
+# the library. This is probably just me not understanding linker syntax well
+# enough, but after this rename it works. If someone has a smarter idea,
+# I am all ears.
+
+cp $root_dir/XClib/xc_lib.a $root_dir/XClib/libqexc.a
```

Comment:
I know that this solution is hacky, but I couldn't find a better. Since you wrote most of the script @elcorto , do you maybe have an idea? The problem is that with the original name of the XClib library file, I cannot make the linker find it, so I renamed it to adhere to the standard naming scheme.

Reply:
So the linker doesn't pick it up if it is not named `lib*.a`? Since it is a static lib you should be able to pass it directly as `$root_dir/XClib/xc_lib.a` to `f2py` as we did with `$root_dir/clib/clib.a` in the old version of the script.

Reply:
Yes, exactly, the linker doesn't pick it up properly. I will try your suggestion!

Reply:
Tried it, it worked. Thanks!